test/ruby/signature/test_test.rb

```diff
@@ -148,6 +148,9 @@ def self.name
   def test_type_check
     SignatureManager.new do |manager|
       manager.files[Pathname("foo.rbs")] = <<EOF
+class Array[Elem]
+end
```

Comment:
`Array` is not defined by SignatureManager in test_helper.
So the test failed by this patch.